code/go/internal/validator/semantic/validate_dimensions.go

```diff
@@ -17,6 +17,11 @@ func ValidateDimensionFields(pkgRoot string) errors.ValidationErrors {
 }
 
 func validateDimensionField(fieldsFile string, f field) errors.ValidationErrors {
+	if f.External != "" {
+		// External fields can be used as dimensions, but we cannot resolve
```

Comment:
At some point in the future, it will be necessary. Of course, we aren't there yet, not even thinking about other implementations of validators (for example JS).

Reply:
Oh, have you said JS? https://github.com/elastic/package-spec/pull/245/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R23 :sweat_smile:

Reply:
I'm losing trust in the dependency graph :D